src/commoncode/functional.py

```diff
@@ -53,7 +61,7 @@ def flatten(seq):
     """
     r = []
     for x in seq:
-        if isinstance(x, (ListType, TupleType)):
+        if isinstance(x, (ListType, TupleType, array)):
```

Comment:
Why did you add array to this? Aldo with my other comment above, using `(list, tuple,)` would be enough IMHO